lxd/api_internal_recover.go

```diff
@@ -96,7 +96,7 @@ func internalRecoverScan(d *Daemon, userPools []api.StoragePoolsPost, validateOn
 		}
 
 		// Load list of project/profile names for validation.
-		profiles, err := tx.GetProfiles(db.ProfileFilter{})
+		profiles, err := dbCluster.GetProfiles(ctx, tx.Tx(), dbCluster.ProfileFilter{})
```

Comment:
The commit message "lxd/storage/volumes/utils: Use new cluster.Profile methods " is incorrect, it covers way more than "lxd/storage/volumes/utils"

Reply:
Yep... I have too much trust in my commit formatting script :)